stylesheets/screens/productslist.css

```diff
@@ -82,25 +73,19 @@
     width: 0.3rem;
 }
 
-.products-container::-webkit-scrollbar {
-    width: 0.5rem;
-}
-
-.filter-options::-webkit-scrollbar-track,
-.products-container::-webkit-scrollbar-track {
+.filter-options::-webkit-scrollbar-track {
     background: var(--grey-bg);
 }
 
-.filter-options::-webkit-scrollbar-thumb,
-.products-container::-webkit-scrollbar-thumb {
+.filter-options::-webkit-scrollbar-thumb {
     background: var(--gold-color);
 }
-  
-@media only screen and (max-width:550px) {
-    .productlist-body {
-        overflow-y: auto;
-    }
-    
+
+.nav-container .btn-modal-alert {
+    display: none;
```

Comment:
can use this as a common utility class in your styles.css file and passing it in the HTML file

Reply:
util class for display requires JS but I dont have time for JS now, so handled using CSS